src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -462,13 +462,6 @@ public static DetailAST parseWithComments(FileContents contents)
     }
 
     @Override
-    public void destroy() {
```

Comment:
This method doesn't do anything if you look further. AbstractCheck#destroy is empty and AbstractFileSetCheck#destroy is also empty. It just creates pitest issues. If you could say what this method should really do, please say it.